### PR TITLE
Fix to correct naming

### PR DESCRIPTION
**Pull request by shopetan**
# WHAT
I found a few minor fixes, so I fixed them in this PR.
This patch does not include anything that needs to be tested. It is just a renaming process. 👍 

I also know at this point that the same fix is needed for other branches. e.g) [permit-plugin: minisched/initialize.go#L110-L114](https://github.com/sanposhiho/mini-kube-scheduler/blob/a8b3100a47ca639a967c8134b1e5033f25e7ecb8/minisched/initialize.go#L110-L114)
It is possible to throw a PR to naive, do you have any ideas?

### CODE REVIEW

File: sched.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func scenario(client clientset.Interface) error {
 		return fmt.Errorf("get pod: %w", err)
 	}
 
-	// pod1 always bound to node1, because node1 got high score from nodenumber score plugin
+	// pod3 always bound to node3, because node1 got high score from nodenumber score plugin
```
Comment (sanposhiho) [inline sched.go]:
```suggestion
	// pod3 always bound to node3, because node3 got high score from nodenumber score plugin
```

**Review by sanposhiho — COMMENTED** [review]
Thanks.

**Comment by sanposhiho** [comment]
@shopetan 

> I also know at this point that the same fix is needed for other branches.

Yes. I'll fix it thanks.

> It is possible to throw a PR to naive, do you have any ideas?

Sorry, what does `native` means?  If it means kubernetes/kubernetes repo, it's not needed. This code did not come from k/k.

**Comment by shopetan** [comment]
@sanposhiho 

> Sorry, what does native means? If it means kubernetes/kubernetes repo, it's not needed. This code did not come from k/k.

Not `native`, it’s `naive` 🙏 .
So "It is possible to throw a PR to naive" means "I can send you a revised similar PR for the other branch" .
Sorry for the confusion. I just wanted to suggest this. 😅

**Pull request merged**